packages/skill-lesson/video/collection/index.tsx

```diff
@@ -792,6 +802,50 @@ const useScrollToActiveLesson = (
   }, [activeElementToScrollTo, scrollContainerRef])
 }
 
+const getLessonChildren = (children: React.ReactNode, props: LessonProps) => {
```

Comment:
It seems like `getLessonChildren` and `getSectionChildren` aren't needed anymore.

I did some refactoring because the rendering was getting weird, such that each section was rendering multiple times. I think that was because we had the root component rendering sections and lessons, but then the nested Section components were using data from context to render themselves additional times.

I shifted some logic down to the `Sections` component to deal with this. Which is why these two functions are no longer being used.

@vojtaholik I'd appreciate if you could advise on these changes since you have a lot more familiarity with this set of components.

![image](https://github.com/skillrecordings/products/assets/694063/b10e7e75-d1a0-45d2-adf8-d32b9db3e798)

Reply:
I haven't looked at the code yet, but purpose of the `getLessonChildren` and `getSectionChildren` was primarily so that we can apply separate styles from the lesson template level.